src/sagemaker/tuner.py

```diff
@@ -760,6 +760,19 @@ def best_estimator(self, best_training_job=None):
         be deployed to an Amazon SageMaker endpoint and return a ``sagemaker.RealTimePredictor``
         object.
 
+        Args:
+            best_training_job (dict): Dictionary containing "TrainingJobName" and
+                "TrainingJobDefinitionName".
+                Example:
+                {
+                "TrainingJobName": "my_training_job_name",
+                "TrainingJobDefinitionName": "my_training_job_definition_name"
+                }
```

Comment:
have you checked how this formats when rendered? My hunch is that you need more newlines and indentation.

Reply:
Fixed! Good callout.

Reply:
![image](https://user-images.githubusercontent.com/51346608/76108047-b4b55980-5f8e-11ea-90b0-f80ea780ed5e.png)

Reply:
7th time's the charm
![image](https://user-images.githubusercontent.com/51346608/76109222-e16a7080-5f90-11ea-970e-97a0a6c9abce.png)